lte/gateway/c/core/oai/tasks/nas5g/src/ies/M5GDNN.cpp

```diff
@@ -44,6 +44,14 @@ int DNNMsg::DecodeDNNMsg(DNNMsg* dnn_message, uint8_t iei, uint8_t* buffer,
   uint8_t dnn_length = 0;
   uint8_t dnn_len = 0;
   DECODE_U8(buffer + decoded, dnn_len, decoded);
+
+  if (ielen <= dnn_len) {
+    OAILOG_ERROR(LOG_NAS5G,
+                 "Mismatch Length: IE length : %d, DNN String length: %d",
```

Comment:
Both variables are of type, unit8_t. Change the format specifier to %u

Reply:
Addressed Review Comment